src/clients/secretsManagerService.js

```
@@ -0,0 +1,92 @@
+import {
+  SecretsManagerClient,
+  CreateSecretCommand,
```

ESLint

[error] Member 'CreateSecretCommand' of the import declaration should be sorted alphabetically.

(sort-imports)